src/packages/rx-recompose/createEventHandler.js

```diff
@@ -1,21 +1,18 @@
-import { Subject } from 'rx'
+import { Observable } from 'rx'
 
-// Idea and implementation borrowed from
-// https://github.com/fdecampredon/rx-react
 const createEventHandler = () => {
-  function subject(value) {
-    subject.onNext(value)
+  const observers = []
```

Comment:
Just curious, isn't it will be the same just creating single observer, and exporting stream as `stream.publish()` ?